### PR TITLE
"first extension" is a bad title

### PR DESCRIPTION
**Pull request by franzholz**
There has already been the blog example as the first extension.

### CODE REVIEW

File: Documentation/4-FirstExtension/1-the-example-extension.rst [hunk 1]
```diff
@@ -3,11 +3,11 @@
 
 .. include:: /Includes.rst.txt
 
-=====================
-The example extension
-=====================
+=============================
+The store inventory extension
```
Comment (DavidBruchmann) [inline Documentation/4-FirstExtension/1-the-example-extension.rst:7]: In general I think the headline can reflect somehow the difficulty, this is lost when it's getting just the extension name which doesn't reflect this.

Comment (franzholz) [reply Documentation/4-FirstExtension/1-the-example-extension.rst:7]: There is more than one example extension. I always think about the blog example as the first extension. So this title is misleading and not helpful.

Comment (DavidBruchmann) [reply Documentation/4-FirstExtension/1-the-example-extension.rst:7]: sure, the blog is historically well-known, but to newbies perhaps not.  
Anyway, if you think my comment is not useful, just feel free to ignore it.

**Review by linawolf — APPROVED** [review]

File: Documentation/4-FirstExtension/1-the-example-extension.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/4-FirstExtension/1-the-example-extension.rst:7]: I would agree with @franzholz as there is more then one example extension naming the concrete extension makes navigating easier.

**Pull request merged**